test/test_apps/shared/test/page_objects/books/dummy_show_page_with_unresolvable_path.rb

```diff
@@ -0,0 +1,11 @@
+module PageObjects
+  module Books
+    class DummyShowPageWithUnresolvablePath < AePageObjects::Document
+      # This page is used to test when the first path is correct, and the second path is incorrect.
+      # In the past, the code will blow up with an unreadable error that doesn't indicate that the
+      # failure is due to the PathNotResolvable. We now tests that it has a understandable error message.
+      path :book
```

Comment:
That's a good call: I took a deeper look at it, and it can be reproduced without the first path.
I thought it was needed because there was another way to reproduce the same error, but it's harder to write a test for that.

The alternative way I used was:
1. `DummyPage.visit`
2. PageObject will use the first path to visit the page
3. The browser gets redirected to another page that doesn't match the first path
4. Exception with unclear message